storm-server/src/main/java/org/apache/storm/container/oci/OciContainerManager.java

```diff
@@ -118,6 +118,8 @@ public void reserveResourcesForWorker(String workerId, Integer workerMemoryMb, I
     public void releaseResourcesForWorker(String workerId) {
         workerToCpu.remove(workerId);
         workerToMemoryMb.remove(workerId);
+        workerToCores.remove(workerId);
```

Comment:
if for some reason this PR is not committed, this looks like a separate PR that should be addressed to get in